.github/workflows/update-charts.yml

```diff
@@ -0,0 +1,126 @@
+name: Update helm charts
+
+on:
+  repository_dispatch:
+    types: [update-chart]
+
+jobs:
+  check-update-type:
+    name: Detect update type
+    runs-on: ubuntu-latest
+    outputs:
+      update_type: ${{ steps.step1.outputs.update_type }}
+    steps:
+      - name: Check if it is patch update
+        id: step1
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let oldVersion = context.payload.client_payload.oldVersion.split(".")
+            let version = context.payload.client_payload.version.split(".")
+            if (oldVersion.length != 3 && version.length != 3) {
+                    core.setFailed("Invalid version format")
+            }
+            if (oldVersion[0] != version[0]){
+                    core.setOutput("update_type", "major")
+
+            } else if(oldVersion[1] != version[1]){
+                    core.setOutput("update_type", "minor")
+
+            } else if(oldVersion[2] != version[2]){
+                    core.setOutput("update_type", "patch")
+
+            } else {
+                    core.setFailed("Invalid version format")
```

Comment:
I think this happens if there's no difference between the old version and the new one

Reply:
I agree that semver can be more complicated that and sooner than later people will want to add metadata information.
Another library https://github.com/Masterminds/semver

Reply:
Fair enough. I changed job to use the script shared by Flavio. But I would be great create a CLI using the Mastermind/semver.